modules/gles31/functional/es31fProgramInterfaceQueryTests.cpp

```diff
@@ -5590,13 +5596,15 @@ static void generateProgramInputLocationBlockContents (Context& context, const R
 
 static void generateProgramOutputLocationBlockContents (Context& context, const ResourceDefinition::Node::SharedPtr& parentStructure, tcu::TestCaseGroup* targetGroup, deUint32 presentShadersMask, bool isGL45)
 {
+	DE_UNREF(isGL45);
 	const bool									inDefaultBlock	= parentStructure->getType() == ResourceDefinition::Node::TYPE_DEFAULT_BLOCK;
 	const ResourceDefinition::Node::SharedPtr	output			= (inDefaultBlock)
 																	? (ResourceDefinition::Node::SharedPtr(new ResourceDefinition::StorageQualifier(parentStructure, glu::STORAGE_OUT)))
 																	: (parentStructure);
 	const glu::ShaderType						lastStage		= getShaderMaskLastStage(presentShadersMask);
 
-	const bool									inBlockArray	= 0 == deStringEqual("block_array", targetGroup->getName());
+	const bool									inBlockArray	= DE_TRUE == deStringEqual("block_array", targetGroup->getName());
```

Comment:
Except disabled tests mentioned in the description it also enables the following tests:
```
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.vertex_fragment.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.vertex_fragment.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_vertex.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_vertex.var_struct_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_vertex.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_fragment.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_fragment.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_tess_eval.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_tess_eval.var_struct_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_tess_eval.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_geometry.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_geometry.var_struct_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.separable_geometry.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.out.named_block.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.out.named_block.var_struct_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.out.named_block.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.out.named_block_explicit_location.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.out.named_block_explicit_location.var_struct_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.out.named_block_explicit_location.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.patch_out.named_block.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.patch_out.named_block.var_struct_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.patch_out.named_block.var_array_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.patch_out.named_block_explicit_location.var_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.patch_out.named_block_explicit_location.var_struct_explicit_location
TEST: dEQP-GL45.functional.program_interface_query.program_output.location.interface_blocks.patch_out.named_block_explicit_location.var_array_explicit_location
```
According to the variable name I suspect that it was supposed to check if current group is `block_array`. (Note: `deStringEqual` returns True if strings equal, false otherwise.)